R/translations.R

```diff
@@ -301,6 +311,12 @@ translations_indirect_generic <- list(
   },
   ifnull = function(x, y) {
     eval(substitute(quote(ifelse(is.na(x), y, x))))
+  },
+  isnull = function(x, y) {
+    eval(substitute(quote(ifelse(is.na(x), y, x))))
+  },
```

Comment:
Uh oh—`isnull()` means something different than this in MySQL and Hive: it takes only one argument and returns true or false. Can you handle both cases in this definition?

Reply:
Missed this one originally. I checked `nargs()` to determine the correct R translation.